### DISCUSSION TITLE
was working now it's not

### DISCUSSION THREAD

**Discussion by Sleia67** [DavidS95/Smokeless_UMAF#43] [General]
I have and 82r4 this was working, but now it's not. My bios updated the other day, and I went to use this to fix everything I had done before, and it doesn't work. I reinstalled it to the formatted USB. Has this happened to anyone and does anyone know a possible fix? Thank you for your time!

**Comment by Sleia67** [comment]
Solved secure boot got turned back on

**Comment by jhemmond** [comment]
How did you solve the Secure Boot constantly going back on?
